lingua_franca/lang/common_data_fa.py

```diff
@@ -0,0 +1,115 @@
+#
+# Copyright 2017 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from collections import OrderedDict
+from .parse_common import invert_dict
+
+_FUNCTION_NOT_IMPLEMENTED_WARNING = "تابع خواسته شده در زبان فارسی پیاده سازی نشده است."
+
+
+_FRACTION_STRING_FA = {
+    2: 'دوم',
+    3: 'سوم',
+    4: 'چهارم',
+    5: 'پنجم',
+    6: 'ششم',
+    7: 'هفتم',
+    8: 'هشتم',
+    9: 'نهم',
+    10: 'دهم',
+    11: 'یازدهم',
+    12: 'دوازدهم',
+    13: 'سیزدهم',
+    14: 'چهاردهم',
+    15: 'پونزدهم',
+    16: 'شونزدهم',
+    17: 'هیفدهم',
+    18: 'هیجدهم',
+    19: 'نوزدهم',
+    20: 'بیستم'
+}
+
+
+_farsiOnes = [
```

Comment:
can all these be made UPPERCASE so it's consistent across all languages?

Reply:
Please specify all the names that have problems so I can change all of them at the same time.

Reply:
everything in common_data_XXX essentially, so all dicts and lists in that file

Reply:
Done